.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: go
 go_import_path: pault.ag/go/debian
 go:
+  - 1.9
+  - 1.8
+  - 1.7
   - 1.6
   - 1.5
```

Comment:
It's relatively new, but I'd recommend adding `.x` to all these so you get the latest point release. :+1:

Reply:
See https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use :+1:

Reply:
Nice call! Thanks! Added a `.x` to them all!